taar_lite/recommenders/guid_based_recommender.py

```diff
@@ -61,6 +62,10 @@ def _init_from_ctx(self):
             msg = "Cannot download addon coinstallation file {}".format(ADDON_LIST_KEY)
             self.logger.error(msg)
 
+        # Compute the floor install incidence that recommended addons
+        # must satisfy.  Take 5% of the mean of all installed addons.
+        self._min_installs = np.mean(list(self._guid_rankings.values())) * 0.05
```

Comment:
due to how immensly skewed the addons installation distribution is. I think we should add a logging statement that throws a warning if the absoute number of installed addons defined by this metric drops below 100.